docs/source/intro/tutorial_page.rst

```diff
@@ -0,0 +1,38 @@
+.. _intro:tutorial_page:
+
+**********
+Tutorials
+**********
+
+Welcome to the AiiDA tutorial!
+
+AiiDA has an extensive and well spread tutorials, it's the best to first start the :ref:`Basic tutorial <tutorial:basic>`
+
+
+.. toctree::
+   :maxdepth: 1
+
+   tutorial
```

Comment:
```suggestion
For new users, it is recommended to start with the :ref:`basic tutorial <tutorial:basic>`.
```

Reply:
The toctree is not really helping I find. It is just adding unnecessary links

Reply:
> The toctree is not really helping I find. It is just adding unnecessary links

Well, the idea was to have it ready to go, for adding more tutorials so they would show up in the left panel.  But it's fine we set toctree later, once having more tutorial pages.